pkg/proxy/paths/options/options.go

```diff
@@ -188,5 +187,56 @@ func (o *Options) Merge(o2 *Options) {
 			o.ReqRewriter = o2.ReqRewriter
 		}
 	}
-	o.Custom = strings.Unique(o.Custom)
+	o.Custom = strutil.Unique(o.Custom)
 }
+
+var pathMembers = []string{"path", "match_type", "handler", "methods", "cache_key_params",
+	"cache_key_headers", "default_ttl_secs", "request_headers", "response_headers",
+	"response_headers", "response_code", "response_body", "no_metrics", "collapsed_forwarding",
+	"req_rewriter_name",
+}
+
+// func ProcessTOML() {
```

Comment:
More ded code?

Reply:
this is implemented / cleaned up in a forthcoming PR (see https://github.com/jranson/trickster/blob/2.0-options-cleanup-01/pkg/proxy/paths/options/options.go#L206)